src/Symfony/Component/HttpKernel/Controller/ContainerControllerResolver.php

```diff
@@ -86,6 +88,18 @@ protected function instantiateController($class)
             return $this->container->get($class);
         }
 
-        return parent::instantiateController($class);
+        try {
+            return parent::instantiateController($class);
+        } catch (FatalThrowableError $e) {
+        } catch (\ArgumentCountError $e) {
+        }
+
+        if (null !== $e) {
```

Comment:
$e should be initialized, is it? This "if" should be merged with the next one also.